BlueShift-iOS-SDK/BlueShiftDeepLink.m

```diff
@@ -131,18 +136,20 @@ - (void)deepLinkToPath:(NSArray *)path {
 
 - (UIViewController *)lastViewController {
     // Get the last view controller in the view controller list ...
-    
+    if ([BlueShiftInAppNotificationHelper checkAppDelegateWindowPresent] == NO) {
+        return  nil;
```

Comment:
this nil value is handled at the caller's side, right?

Reply:
Yes, it has been taken care of.